components/about/CustomTable.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="aboutus-table">
+    <tr v-for="(item, index) in listdata" :key="index">
```

Comment:
Codacy has a fix for the issue: [':key' should be on a new line.](https://app.codacy.com/gh/AIESEC-in-Japan/aiesecjp-web/pullRequest?prid=7667176)
```suggestion
    <tr v-for="(item, index) in listdata"
```